### PR TITLE
Short blog post and announcment for 2.5 release

### PR DESCRIPTION
**Pull request by ashb**
Very rough and ready and last minute (sorry!)

~~List of committers is wrong! That was just copied from 2.4~~

Likely rife with spelling mistakes too

### CODE REVIEW

File: landing-pages/site/content/en/blog/airflow-2.5.0/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+This airflow subcommand has been rethought and re-optimized to make it much easier to test your DAGs locally - the major changes are:
+
+a. Task logs are visible right there in the console, instead of hidden away inside the task log files
+b. It is about an order of mangnitude quicker to run the tasks than before (i.e. it gets to running the task code so much quicker); and
+c. Everything runs in one process, so you can put a breakpoint in your IDE, and configure it to run `airflow dags test <mydag>` then debug code!
+
+## Auto tailing task logs in the Grid view
+
+Hopefully the headline point says enough. It's lovely, go check it out.
```
Comment (jedcunningham) [inline landing-pages/site/content/en/blog/airflow-2.5.0/index.md:42]:
```suggestion
Hopefully the headline says enough. It's lovely, go check it out.
```

**Review by jedcunningham — APPROVED** [review]

File: landing-pages/site/content/en/blog/airflow-2.5.0/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+When we released Dataset aware scheduling in September we knew that the tools we gave to manage the Datasets were very much a Minimum Viable Product, and in the last two months the committers and contributors have been hard at work at making the UI much more usable when it comes to Datasets.
+
+But we we aren't done yet - keep an eye out for more improvments coming over the next couple of releases too.
+
+## Greatly improved `airflow dags test` command
+
+This airflow subcommand has been rethought and re-optimized to make it much easier to test your DAGs locally - the major changes are:
+
+a. Task logs are visible right there in the console, instead of hidden away inside the task log files
+b. It is about an order of mangnitude quicker to run the tasks than before (i.e. it gets to running the task code so much quicker); and
```
Comment (blag) [inline landing-pages/site/content/en/blog/airflow-2.5.0/index.md:37]:
```suggestion
b. It is about an order of magnitude quicker to run the tasks than before (i.e. it gets to running the task code so much quicker)
```

File: landing-pages/site/content/en/blog/airflow-2.5.0/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+---
+title: "Apache Airflow 2.5.0: tick-tock"
+linkTitle: "Apache Airflow 2.5.0"
+author: "Ash Berlin-Taylor"
+github: "ashberlin"
+twitter: "ashberlin"
+linkedin: "ashberlin-taylor"
+description: "We're proud to announce that Apache Airflow 2.5.0 has been released many quality of life changes."
```
Comment (blag) [inline landing-pages/site/content/en/blog/airflow-2.5.0/index.md:8]:
```suggestion
description: "We're proud to announce that Apache Airflow 2.5.0 has been released with many quality of life changes."
```

File: landing-pages/site/content/en/announcements/_index.md [hunk 4]
```diff
@@ -13,6 +13,15 @@ menu:
 
 **Note:** Follow [@ApacheAirflow](https://twitter.com/ApacheAirflow) on Twitter for the latest news and announcements!
 
+# December 2, 2022
+
+We’ve just released Apache **Airflow 2.5.0**. You can read all about [what's new in Apache Airflow 2.5.0]({{< ref "blog/airflow-2.5.0/index.md" >}}) blog post.
```
Comment (blag) [inline landing-pages/site/content/en/announcements/_index.md:18]: Removing the link here to make the sentence more visible:

> You can read all about what's new in Apache Airflow 2.5.0 blog post.

Seems like that's missing an article somewhere, no?

How's this:

> You can read all about it in our what's new in Apache Airflow 2.5.0 blog post.

(Turning that into a suggestion to make it easier to incorporate)
```suggestion
We’ve just released Apache **Airflow 2.5.0**. You can read all about it in our [what's new in Apache Airflow 2.5.0]({{< ref "blog/airflow-2.5.0/index.md" >}}) blog post.
```

File: landing-pages/site/content/en/blog/airflow-2.5.0/index.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+---
+title: "Apache Airflow 2.5.0: tick-tock"
```
Comment (blag) [inline landing-pages/site/content/en/blog/airflow-2.5.0/index.md:2]:
```suggestion
title: "Apache Airflow 2.5.0: Tick-Tock"
```

File: landing-pages/site/content/en/blog/airflow-2.5.0/index.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+**Details**:
+
+📦 PyPI: https://pypi.org/project/apache-airflow/2.5.0/ \
+📚 Docs: https://airflow.apache.org/docs/apache-airflow/2.5.0/ \
+🛠️ Release Notes: https://airflow.apache.org/docs/apache-airflow/2.5.0/release_notes.html \
+🐳 Docker Image: docker pull apache/airflow:2.5.0 \
+🚏 Constraints: https://github.com/apache/airflow/tree/constraints-2.5.0
+
+This quicker release cadance is a departure from our previous habit of releasing every five-to-seven months and was a delibarate effort to listen to you, our users, and get the changes and improvments into your workflows sooner.
```
Comment (blag) [inline landing-pages/site/content/en/blog/airflow-2.5.0/index.md:24]:
```suggestion
This quicker release cadence is a departure from our previous habit of releasing every five-to-seven months and was a deliberate effort to listen to you, our users, and get the changes and improvements into your workflows earlier.
```

File: landing-pages/site/content/en/blog/airflow-2.5.0/index.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+## Usability improvements to the Datasets UI
+
+When we released Dataset aware scheduling in September we knew that the tools we gave to manage the Datasets were very much a Minimum Viable Product, and in the last two months the committers and contributors have been hard at work at making the UI much more usable when it comes to Datasets.
+
+But we we aren't done yet - keep an eye out for more improvments coming over the next couple of releases too.
```
Comment (blag) [inline landing-pages/site/content/en/blog/airflow-2.5.0/index.md:30]:
```suggestion
But we we aren't done yet - keep an eye out for more improvements coming over the next couple of releases too.
```

File: landing-pages/site/content/en/blog/airflow-2.5.0/index.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+b. It is about an order of mangnitude quicker to run the tasks than before (i.e. it gets to running the task code so much quicker); and
+c. Everything runs in one process, so you can put a breakpoint in your IDE, and configure it to run `airflow dags test <mydag>` then debug code!
+
+## Auto tailing task logs in the Grid view
+
+Hopefully the headline point says enough. It's lovely, go check it out.
+
+## More improvments to Dynamic-Task mapping
+
+In a similar vein to the improvments to the Dataset (UI), we have continued to iterate on and improve the feature we first added in Airflow 2.3, Dynamic Task Mapping, and 2.5 includes [dozens of improvements](https://github.com/apache/airflow/pulls?q=is%3Apr+author%3Auranusjr+is%3Aclosed+milestone%3A%22Airflow+2.5.0%22).
```
Comment (blag) [inline landing-pages/site/content/en/blog/airflow-2.5.0/index.md:46]:
```suggestion
In a similar vein to the improvements to the Dataset (UI), we have continued to iterate on and improve the feature we first added in Airflow 2.3, Dynamic Task Mapping, and 2.5 includes [dozens of improvements](https://github.com/apache/airflow/pulls?q=is%3Apr+author%3Auranusjr+is%3Aclosed+milestone%3A%22Airflow+2.5.0%22).
```

File: landing-pages/site/content/en/blog/airflow-2.5.0/index.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+
+## Thanks to the contributors
+
+ Andrey Anshin, Ash Berlin-Taylor, blag, Bolke de Bruin, Brent Bovenzi, Chenglong Yan, Daniel Standish, Dov Benyomin Sohacheski, Elad Kalif, Ephraim Anierobi, Jarek Potiuk, Jed Cunningham, Jorrick Sleijster, Michael Petro, Niko, Pierre Jeambrun, Tzu-ping Chung and many more, over 75 of you. Thank you!
```
Comment (blag) [inline landing-pages/site/content/en/blog/airflow-2.5.0/index.md:51]:
```suggestion
Andrey Anshin, Ash Berlin-Taylor, blag, Bolke de Bruin, Brent Bovenzi, Chenglong Yan, Daniel Standish, Dov Benyomin Sohacheski, Elad Kalif, Ephraim Anierobi, Jarek Potiuk, Jed Cunningham, Jorrick Sleijster, Michael Petro, Niko, Pierre Jeambrun, Tzu-ping Chung and many more, over 75 of you. Thank you!
```

File: landing-pages/site/content/en/blog/airflow-2.5.0/index.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+And especially thank you to Ephraim who tireleslly works behind the scenes as release manager!
```
Comment (blag) [inline landing-pages/site/content/en/blog/airflow-2.5.0/index.md:53]:
```suggestion
And a special thank you to Ephraim who tirelessly worked behind the scenes as release manager!
```

**Review by blag — COMMENTED** [review]
Minor corrections, otherwise looks good.

**Review by blag — APPROVED** [review]
LGTM

**Pull request merged**